cdap-docs/admin-manual/source/installation/installation-note.txt

```diff
@@ -0,0 +1,18 @@
+.. topic:: Different ways to install CDAP
+
+  There are different ways to install CDAP, depending if it is a distributed or standalone system:
+
+  - For **CDH (Cloudera)**, you can follow :ref:`these instructions <cloudera-configuring>` using Cloudera Manager.
+
+  - For **HDP (Hortonworks)**, you can follow :ref:`these instructions <ambari-configuring>` using Apache Ambari.
+
+  - For **MapR**, you can follow :ref:`these notes <mapr-configuring>`.
+
+  - On **other Apache Hadoop distributions**, you can follow the manual method referenced below on this page, 
```

Comment:
Related to above concern: On "unmanaged" CDH, "unmanaged" HDP, or other hadoop distributions (not sure how to say this)